ci/ci/build.py

```diff
@@ -681,11 +681,18 @@ def build(self, batch, code, scope):
                 elif w['kind'] == 'Service':
                     assert w['for'] == 'alive', w['for']
                     port = w.get('port', 80)
+                    resource_type = w.get('resource_type', 'deployment')
                     timeout = w.get('timeout', 60)
+                    if resource_type == 'statefulset':
+                        wait_cmd = f'kubectl -n {self.namespace} wait --timeout=1h --for=condition=ready pods --selector=app={name}'
```

Comment:
should we just allow the full selector to be used in build.yaml? I could imagine a selector that is "foo={name}"